websites/C/ClarityClan/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Leonardo Jacon#0001",
+      "id": "600148454668369930"
+    },
+    "url": "clarityclan.team",
+    "description": {
+      "en": "Clarity Clan is a Minecraft Clan, growing in the community of MCC."
+    },
+    "service": "Clarity Clan",
```

Comment:
The service name must be the same as your directory name. Either change this to `Clarity Clan` or change the name of the directory.